c2corg_ui/templates/index.html

```diff
@@ -1,4 +1,48 @@
 <%inherit file="base.html"/>
 
-<h1 translate class="welcome-title text-center">Welcome to Camptocamp.org</h1>
-<app-map></app-map>
+<div class="home-page">
+  <section ng-hide="userCtrl.auth.isAuthenticated()" class="site-intro not-logged">
+    <div class="intro-banner">
+      <h1 class="welcome-title text-center" translate>The mountain sports community</h1>
+
+      <div class="route-activities text-center">
+        <span class="activity icon-hiking"></span>
+        <span class="activity icon-skitouring"></span>
+        <span class="activity icon-mountain_biking"></span>
+      </div>
+
+      <p class="text-center" translate>Camptocamp.org aims to
+        facilitate information sharing between
+        mountain addicts and contribute to the safety of mountain activities.
+      </p>
+
+      <div class="intro-shortcuts">
+        <a href="${request.route_url('routes_index')}">
+          <button class="btn green-btn">
+            <span class="glyphicon glyphicon-arrow-right"></span>
+            <span translate>Topoguide</span><br>
+            <span translate class="small">Cartes, itinéraires, sommets...</span>
+          </button>
+        </a>
+        <a href="${discourse_url}">
+          <button class="btn blue-btn" translate>
+            <span class="glyphicon glyphicon-th-list"></span>
+            <span translate>Forum</span><br>
+            <span translate class="small">Partagez avec nous!</span>
+          </button>
+        </a>
+      </div>
+
+    </div>
+  </section>
+  <section ng-hide="!userCtrl.auth.isAuthenticated()" class="site-intro">
+    <div class="intro-banner">
+      <h1 class="welcome-title text-center" translate>Feed</h1>
```

Comment:
Yes, I mean "Feed" => "Activity feed".
It's more less generic and i'ts better to change it before a translation is added in Transifex (because I think that the translations are lost in Transifex when updating the source string!?). Since the PR has been merged, I think you have to make a new one.